src/protocol/definitions/ping360.json

```diff
@@ -187,17 +197,109 @@
                         "description": "Head angle in gradians (0 to 399 = 0 to 360 degrees)"
                     }
                 ]
+            },
+            "forwards_upload_data": {
+                "id": "2305",
+                "description": "Follow `stationary_upload_data (2303)` but with forward steps.",
+                "payload": [
+                    {
+                        "name": "mode",
+                        "type": "u8",
+                        "description": "Operating mode (1 for Ping360)"
+                    },
+                    {
+                        "name": "gain_index",
+                        "type": "u8",
+                        "description": "Analog gain index (0 = low, 1 = normal, 2 = high)"
+                    },
+                    {
+                        "name": "angle",
+                        "type": "u16",
+                        "description": "Head angle in gradians (3 ASCII/Hex digits 0 to 399 decimal = 0 to 360 degrees)"
+                    },
+                    {
+                        "name": "pulse_usec",
+                        "type": "u16",
+                        "description": "Tx pulse length in microseconds. (1 to 1000)"
+                    },
+                    {
+                        "name": "interval",
+                        "type": "u16",
+                        "description": "Sample interval in 25nsec clock ticks (40 to 40000 == 1 microsecond to 1000 microseconds)"
+                    },
+                    {
+                        "name": "num_points",
```

Comment:
you should remove the 'num_points' field, it is automatically inserted by the generator python (see the generated output).

Reply:
Fixed